composer.json

```diff
@@ -70,7 +71,7 @@
             "@ci:tests"
         ],
         "ci:php:fixer": "\"./.phive/php-cs-fixer\" --config=config/php-cs-fixer.php fix --dry-run -v --show-progress=dots --diff bin src tests config",
-        "ci:php:lint": "find src tests config bin -name '*.php' -print0 | xargs -0 -n 1 -P 4 php -l",
+        "ci:php:lint": "\"./vendor/bin/parallel-lint\" src tests config bin",
```

Comment:
```suggestion
        "ci:php:lint": "parallel-lint src tests config bin",
```

Reply:
This works for me.  I think it used not to.  I just copied what was in Emogrifier.  I guess Composer has been updated not to require a specific path to the `bin` directory.